content/storage/mysql/docker-compose-for-mysql.md

```diff
@@ -71,6 +73,13 @@ services:
     networks:
       - compose_network
 
+# https://serverfault.com/questions/871090/how-to-use-docker-secrets-without-a-swarm-cluster
+secrets:
+  mysql_root_password:
+    file: ./mysql_root_password.txt
+  mysql_password:
+    file: ./mysql_password.txt
```

Comment:
I used to have a secret file problem on Docker for Windows.
However, it was long time ago, I will check if it work without any issues.

Reply:
@aaronamm For me I never see a secret file on Docker for Windows. However, In order to use Docker on Window, it's required to install WSL, I think with Linux environment, it will be fine.